tests/e2e/module_templates/moduletemplate_template_operator_regular_warning.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: operator.kyma-project.io/v1beta2
+kind: ModuleTemplate
+metadata:
+  name: template-operator-regular
+  namespace: kcp-system
+  labels:
+    "operator.kyma-project.io/module-name": "template-operator"
+  annotations:
+    "operator.kyma-project.io/is-cluster-scoped": "false"
+spec:
+  channel: regular
+  data:
+    apiVersion: operator.kyma-project.io/v1alpha1
+    kind: Sample
+    metadata:
+      name: sample-yaml
+    spec:
+      resourceFilePath: "./module-data/yaml"
+  descriptor:
+    component:
+      componentReferences: []
+      labels:
+        - name: security.kyma-project.io/scan
+          value: enabled
+          version: v1
+      name: kyma-project.io/template-operator
+      provider: internal
+      repositoryContexts:
+        - baseUrl: europe-west3-docker.pkg.dev/sap-kyma-jellyfish-dev/template-operator
+          componentNameMapping: urlPath
+          type: ociRegistry
+      resources:
+        - access:
+            digest: sha256:e88f3d1a80dbce2084f4991905a3e9847a58bd0e5b666754aa9105b9d49e82ba
+            type: localOciBlob
```

Comment:
this module template is not generated with latest cli, the type should be `ociblob`.